File: RegistrationApi/Models/Person.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace RegistrationApi.Models
{
    public class Person
    {
        public Guid Id { get; set; }
        [MaxLength(50)]
        [Column(TypeName = "varchar")]
        public string FullName { get; set; }
        [MaxLength(50)]
        [Column(TypeName = "varchar")]
        public string CPF { get; set; }
        [MaxLength(50)]
        [Column(TypeName = "varchar")]
        public string Email { get; set; }
        public virtual Address Address { get; set; }
        public Guid AddressId { get; set; }
        public virtual Company Company { get; set; }
        public Guid CompanyId { get; set; }
    }
}

File: RegistrationApp/src/app/models/Person.ts
export interface Person{
  id: string;
  fullName:string;
  cpf:string;
  email:string;
  addressId:string;
  companyId:string;
  address:{
    id:string;
    streetName:string;
    number:number;
    district:string;
    zipCode:string;
  };
  company:{
    id:string;
    companyName:string;
    cnpj:string;
    address:{
      id:string;
      streetName:string;
      number:number;
      district:number;
      zipCode:string;
    }
  }
}
